abb_irb6600_support/urdf/irb6600_225_255_macro.xacro

```diff
@@ -0,0 +1,229 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://ros.org/wiki/xacro">
+  <xacro:include filename="$(find abb_resources)/urdf/common_materials.xacro"/>
+  <xacro:macro name="abb_irb6600_225_255" params="prefix">
+    <!-- link list -->
+    <link name="${prefix}base_link">
+      <visual>
+        <origin xyz="0 0 0" rpy="0 0 0" />
+        <geometry>
+          <mesh filename="package://abb_irb6600_support/meshes/irb6600_225_255/visual/base_link.dae" />
+        </geometry>
+      </visual>
+      <collision>
+        <origin xyz="0 0 0" rpy="0 0 0" />
+        <geometry>
+          <mesh filename="package://abb_irb6600_support/meshes/irb6600_225_255/collision/base_link.stl" />
+        </geometry>
+        <xacro:material_abb_yellow />
+      </collision>
+    </link>
+    <link name="${prefix}link_1">
+      <visual>
+        <origin xyz="0 0 0" rpy="0 0 0" />
+        <geometry>
+          <mesh filename="package://abb_irb6600_support/meshes/irb6600_225_255/visual/link_1.dae" />
+        </geometry>
+      </visual>
+      <collision>
+        <origin xyz="0 0 0" rpy="0 0 0" />
+        <geometry>
+          <mesh filename="package://abb_irb6600_support/meshes/irb6600_225_255/collision/link_1.stl" />
+        </geometry>
+        <xacro:material_abb_yellow />
+      </collision>
+      <collision>
+        <origin xyz="0 0 0" rpy="0 0 0" />
+        <geometry>
+          <mesh filename="package://abb_irb6600_support/meshes/irb6600_225_255/collision/link_1_mtr.stl" />
+        </geometry>
+        <xacro:material_abb_yellow />
+      </collision>
+      <collision>
+        <origin xyz="0 0 0" rpy="0 0 0" />
+        <geometry>
+          <mesh filename="package://abb_irb6600_support/meshes/irb6600_225_255/collision/link_1_mtr2.stl" />
+        </geometry>
+        <xacro:material_abb_yellow />
+      </collision>
+      <collision>
+        <origin xyz="0 0 0" rpy="0 0 0" />
+        <geometry>
+          <mesh filename="package://abb_irb6600_support/meshes/irb6600_225_255/collision/link_1_wt.stl" />
```

Comment:
It looks like I may not be able to add the mimic joints. I can not locate the CAD model on ABB's website, it may be because they no longer offer this model.